src/runtime/graph_executor/graph_executor.cc

```diff
@@ -358,11 +415,16 @@ void GraphExecutor::SetupStorage() {
 void GraphExecutor::SetupOpExecs() {
   op_execs_.resize(this->GetNumOfNodes());
   input_dltensors_.resize(num_node_entries());
+  output_dltensors_.resize(num_node_entries());
   std::unordered_set<uint32_t> input_node_eids;
   for (size_t i = 0; i < input_nodes_.size(); i++) {
     uint32_t nid = input_nodes_[i];
     input_node_eids.insert(entry_id(nid, 0));
   }
+  std::unordered_set<uint32_t> output_node_id;
+  for (size_t i = 0; i < outputs_.size(); i++) {
+    output_node_id.insert(outputs_[i].node_id);
```

Comment:
should here use entry_id?

Reply:
Here "output_node_id" is used to determine whether it is an output node by nid

Reply:
nid -> eid